src/ImageProcessorCore/Formats/Tiff/TiffExifExtractor.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace ImageProcessorCore.Formats
+{
+    /// <summary>
+    ///
+    /// </summary>
+    internal class TiffExifExtractor : ITiffVisitor
+    {
+        private readonly Dictionary<ushort, TiffTag> _exifTags; 
+
+        public List<ImageProperty> Properties { get; private set; }
+
+        public TiffExifExtractor()
+        {
+            Properties = new List<ImageProperty>();
+            
+            _exifTags = TiffTagRegistry.Instance.Tags
+                .Where( i => i.TagGroup == "Exif")
+                .ToDictionary(i => i.TagId);
+
+        }
+
+        public void Visit(TiffProperty property)
+        {
+            if (_exifTags.ContainsKey(property.Tag.TagId))
+            {
+                // until the image property can hold more than just a string value....
+                ImageProperty imageProperty = new ImageProperty(property.Tag.Name, property.Value.ToString());
```

Comment:
Hmm... Adding a type/value property would certainly by easy to do and it might be a good first step. However, I think it will be really difficult for the user to understand and use the values. It would take a lot of tag knowledge to know how to translate the values. Take an example

**The Orientation Tag**: 0x112

The value is an integer between 1 and 8
But the meaning of those values are as follows...

1 = 0th row at top, 0th column at left
2 = 0th row at top, 0th column at right
3 = 0th row at bottom, 0th column at right
4 = 0th row at bottom, 0th row at left
5 = 0th row at left, 0th column at top
6 = 0th row at right, 0th column at top
7 = 0th row at right, 0th column at bottom
8 = 0th row at left, 0th column at bottom

To make it easy, I would think we would need to translate that to a documented enumeration on a strongly typed EXIF structure. The values that we do not translate into the EXIF structure could still exists in a generic list of ImageProperty. That way they will still get access to them if needed.